api/fbs/v1/schema.fbs

```diff
@@ -0,0 +1,121 @@
+// Licensed to Apache Software Foundation (ASF) under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Apache Software Foundation (ASF) licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+include "database.fbs";
+
+namespace banyandb.v1;
+
+// TraceSeries represents a trace storage object
+table TraceSeries {
+    // metadata is the identity of a trace series
+    metadata:Metadata;
+    // duration determines how long a TraceSeries keeps its data
+    // unit    = "ns" | "ms" | "s" | "m" | "h" | "d" | "w" .
+    // duration: int_literal unit
+    duration:string;
```

Comment:
Do we need a data structure for duration? Otherwise, we have to parse it everytime.

Reply:
Make sense. A parsed result might be more reasonable here.

Reply:
done